.github/CODEOWNERS

```diff
@@ -75,6 +75,7 @@
 /x-pack/plugins/ingest_manager/ @elastic/ingest
 /x-pack/legacy/plugins/ingest_manager/ @elastic/ingest
 /x-pack/plugins/observability/ @elastic/logs-metrics-ui @elastic/apm-ui @elastic/uptime @elastic/ingest
```

Comment:
I don't know how much they (will) overlap, but maybe it makes sense to also add stack-monitoring-ui as a code owner for the shared observability plugin.

Reply:
`stack-monitoring-ui` is a [subteam](https://github.com/orgs/elastic/teams/logs-metrics-ui/teams) of `logs-metrics-ui`, so they will get notified.